resources/testdriver.js

```diff
@@ -49,6 +49,10 @@
      * @namespace {test_driver}
      */
     window.test_driver = {
+        get event_target() {
+            return window.test_driver_internal.event_target;
```

Comment:
just an idea, let's make test_driver an instance of a class that extends EventTarget?

Reply:
AFAIU `test_driver` is implementation-dependent, and should throw an exception if the functionality is not supported for whatever reason. If follow this combined with your suggestion, I'd expose this 2 mehtods instead:
* [EventTarget.addEventListener()](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener)
Registers an event handler of a specific event type on the EventTarget.
* [EventTarget.removeEventListener()](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener)
Removes an event listener from the EventTarget.